src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,197 @@
+        String atTime = info[1];
+        tasks.add(new Event(taskDescription, atTime));
+        System.out.println("Got it. I've added this task: ");
+        System.out.println(tasks.get(tasks.size() - 1).toString());
+        System.out.println("Now you have " + tasks.size() + " tasks in the list.");
+    }
+
+    private static void addDeadline(String line) {
+        String[] words = line.split(" ", 2);
+        String detailWords = words[1];
+        String[] info = detailWords.split(" /by ", 2);
+        String taskDescription = info[0];
+        String byDate = info[1];
+        tasks.add(new Deadline(taskDescription, byDate));
+        System.out.println("Got it. I've added this task: ");
+        System.out.println(tasks.get(tasks.size() - 1).toString());
+        System.out.println("Now you have " + tasks.size() + " tasks in the list.");
+    }
+
+    private static void addTodo(String line) {
+        String[] words = line.split(" ", 2);
+        String taskDescription = words[1];
+        tasks.add(new Todo(taskDescription));
+        System.out.println("Got it. I've added this task: ");
+        System.out.println(tasks.get(tasks.size() - 1).toString());
+        System.out.println("Now you have " + tasks.size() + " tasks in the list.");
+    }
+
+    private static void markDone(String line) {
+        String[] words = line.split(" ", 2);
+        int doneIndex = Integer.parseInt(words[1]);
+        Task doneTask = tasks.get(doneIndex - 1);
+        doneTask.markAsDone();
+        System.out.println("Nice! I've marked this task as done: ");
+        System.out.println(doneTask.toString());
+    }
+
+    private static void printList() {
+        System.out.println("Here are the tasks in your list: ");
+        int index = 0;
+        while (index < tasks.size()) {
+            System.out.println((index + 1) + "." + tasks.get(index).toString());
+            index++;
+        }
+    }
+
+    public static void printBye() {
+        System.out.println("____________________________________________________________\n" +
+                " Bye. Hope to see you again soon!\n" +
+                "____________________________________________________________\n");
+    }
+
+    private static void printGreeting() {
+        System.out.println("Hello from\n" + Duke.LOGO);
+        System.out.println("____________________________________________________________\n" +
+                " Hello! I'm Duke\n" +
+                " What can I do for you?\n" +
+                "____________________________________________________________\n");
+    }
+
+    private static void loadTasks() throws IOException {
+        File f = new File("data/duke.txt");
+        f.getParentFile().mkdir();
+        f.createNewFile();
+        Scanner s = new Scanner(f);
+        while (s.hasNext()) {
+            loadData(s.nextLine());
+        }
+    }
+
+    private static void loadData(String nextLine) {
+        if (nextLine.contains("T")) {
```

Comment:
Again, what if the saved line in the file has more identifiers? like a line could have "T" and "D" at the same time.